infrastructure/run_training_docker.py

```diff
@@ -171,22 +171,22 @@ def main(syn, args):
             log_text = container.logs()
             create_log_file(log_filename, log_text=log_text)
 
-            # subprocess.check_call(["docker", "exec", "logging", "mkdir",
-            #                        "logs/" + str(args.submissionid)])
-            # subprocess.check_call(["docker", "cp", os.path.abspath(log_filename),
-            #                        "logging:/logs/" + str(args.submissionid) + "/"])
-
-            # subprocess.check_call(["docker", "cp", os.path.abspath(stats_log),
-            #                        "logging:/logs/" + str(args.submissionid) + "/"])
-
+            subprocess.check_call(["docker", "exec", "logging", "mkdir",
```

Comment:
Since you made the folder here, will this fail during inference step? Can you submit the baseline to a TEST queue to validate this?